nipoppy/workflow/proc_pipe/tractoflow/run_tractoflow.py

```diff
@@ -472,14 +473,15 @@ def run(participant_id, global_configs, session_id, output_dir, use_bids_filter,
         tdir = np.unique(tvec, axis=0)
 
         ## compute and print the maximum shell
-        plmax = int(np.floor((-3 + np.sqrt(1 + 8 * tdir.shape[1])) / 2.0))
+        plmax = int(np.floor((-3 + np.sqrt(1 + 8 * tdir.shape[1]) / 2.0)))
```

Comment:
I think we need a bit more detail / explanation on this formula.

Reply:
I've had a hard time finding the reference, but I can clarify it. 

It identifies the highest lmax possible based on the number of directions (`tdir.shape[1]`).

If you try and fit an lmax higher than is supported, it will be under specified in it's estimates if it runs at all.